src/components/TierPurchase/TierPurchase.js

```diff
@@ -12,142 +12,104 @@ import { func, instanceOf, node, number, string } from 'prop-types'
 import FarmhandContext from '../Farmhand/Farmhand.context'
 import './TierPurchase.sass'
 
-export class TierPurchase extends Component {
-  state = {
-    selectedTier: '',
-  }
+export function TierPurchase({
```

Comment:
YAY! Thank you for converting this to a functional component. I'd like to convert all of Farmhand's components to functional components!